app/pods/agenda/route.js

```diff
@@ -20,7 +19,7 @@ export default class AgendaRoute extends Route {
     const agenda = await this.store.findRecord('agenda', agendaId, {
       reload: 'true',
     });
-    const reverseSortedAgendas = await this.store.query('agenda', {
+    const reversedAgendas = await this.store.query('agenda', {
```

Comment:
`reverseSortedAgendas` vs `reversedAgendas`. I must say I found the original name clearer

Reply:
Initially the full name was more clear.
When I started using it on multiple components, the name seemed too long and knowing the contents it seemed to make sense to shorten it.
For clarity, will rename all references (36) back to the full name